UPGRADING.md

```diff
@@ -0,0 +1,24 @@
+# Upgrading
+
+This section describes how to upgrade a conjur Server.
+
+## Basic upgrade
+
+The following steps should be done on each upgrade:
+1. Edit `docker-compose.yml` conjur service image tag to {x+1}
+2. Delete current conjur container: 
+`docker rm -f conjur`
+3. Rerun docker-compose:
+`docker-compose up -d`
+4. View docker containers and verify all are healthy, up and running:
+ `docker ps -a`
+
+* side note, it is possible you will need to reassign `CONJUR_DATA_KEY` system variable. Same key as before. 
+`export CONJUR_DATA_KEY="$(< data_key)`
+
+## Specific steps
+
+**This step should be done  when upgrading from version 1.7.2 and below to a newer version**
```

Comment:
@izgeri  just to make sure I understood correctly, also the version should be change to 1.8.0 or 1.7.2 is ok?

Reply:
I am assuming that the FIPS changes will be included in the next tag, and that we'll bump the minor version since we're adding functionality in a backwards compatible manner (following semver - see [here](https://semver.org/))

That would mean they'll be included in 1.8.0

Reply:
that's for sure, what I meant should we say 'from 1.7.2 and below' or 'below version 1.8.0'?
what should be the correct term for this FYO?

Reply:
We could just make it say `Upgrading to 1.8.0`

Reply:
what will happen if a user will upgrade from 1.7.0 to 1.8.1 or even 1.9.0 in the future?
I think it should be clear the any upgrade from a version before 1.8.0 should run this step.
WDYT?
@alexkalish @izgeri

Reply:
I think what you have is fine for now.